packages/nimble-components/src/table/components/cell/styles.ts

```diff
@@ -1,12 +1,17 @@
 import { css } from '@microsoft/fast-element';
-import { display } from '@microsoft/fast-foundation';
 import { standardPadding } from '../../../theme-provider/design-tokens';
 
 export const styles = css`
-    ${display('flex')}
-
     :host {
         padding: 0px calc(${standardPadding} / 2);
+        align-self: center;
+        display: inherit;
```

Comment:
This seems unusual.. why isn't this just display grid / using the display helper used in other components?

Reply:
Changed it to just use `grid`.